src/server/game/Entities/Player/Player.cpp

```diff
@@ -19463,6 +19582,44 @@ void Player::_LoadQuestStatusObjectives(PreparedQueryResult result)
     }
 }
 
+void Player::_LoadQuestStatusObjectiveSpawnTrackings(PreparedQueryResult result)
+{
+
+    ////                                                     0        1        2
+    //QueryResult* result = CharacterDatabase.PQuery("SELECT quest, objective, spawnTrackingId FROM character_queststatus_objectives_spawn_tracking WHERE guid = '{}'", GetGUIDLow());
+
+    if (result)
+    {
+        do
+        {
+            Field* fields = result->Fetch();
+
+            uint32 questID = fields[0].GetUInt32();
+            Quest const* quest = sObjectMgr->GetQuestTemplate(questID);
+            auto itr = m_QuestStatus.find(questID);
```

Comment:
i'd suggest early `continue` if `quest` is null instead of using null for lookup